test: enable golden test for ng_package rule

alexeagle opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

IgorMinar reviewed (commented):
this is failing on CI because the auth setup doesn't work

IgorMinar reviewed (commented):
blocked until we can figure out how to do auth

alexeagle committed:
build: introduce a package.bzl

This lets Angular Bazel users install our transitive deps, rather than have to list them in their WORKSPACE file.
If they want a different version of one of these deps, they just need to install it before calling rules_angular_dependencies.

gregmagolan committed:
build(bazel): use fine-grained npm deps (#26111)

gregmagolan committed:
build: update to rules_typescript 0.20.1 and rules_nodejs 0.15.0 (#26260)

gregmagolan committed:
build(bazel): update to rules_typescript 0.20.2 (#26279)

mary-poppins commented:
You can preview 1f3a31b at https://pr23564-1f3a31b.ngbuilds.io/.

mary-poppins commented:
You can preview fca776c at https://pr23564-fca776c.ngbuilds.io/.

alexeagle committed:
build: update ngcontainer to bazel 0.18.0 (#26465)

* build: update ngcontainer to bazel 0.18.0

* build: update skylint to bazel 0.18

use .bazelignore file to ignore node_modules directory

gregmagolan committed:
build: idiomatic install of @angular/bazel npm package (#26258)

gregmagolan committed:
build: restore integrity lines in yarn.lock

gregmagolan committed:
build: add npm deps after rebase

gregmagolan committed:
build: update to rules_typescript 0.20.3 and rules_nodejs 0.15.1

gregmagolan committed:
build: add comment to /integration/bazel/package.json

alexeagle committed:
refactor(bazel): allow google3 overrides for external deps

alexeagle committed:
ci: don't run google3 tests on bazel external files

alexeagle committed:
style: run buildifier

alexeagle committed:
test: enable ng_package golden test

josephperrott commented:
Closing as this is no longer being worked on.
